python/src/aac/execute/aac_execution_result.py

```diff
@@ -87,12 +87,15 @@ def get_messages_as_string(self) -> str:
         """Return the output messages as a combined string."""
         result = ""
         for message in self.messages:
-            result += message.message + linesep
-            if message.source is not None:
-                result += f"  Source: {message.source.uri}"
-                if message.location is not None:
-                    result += f" (Ln {message.location.line}: Col {message.location.column}: Pos {message.location.position}: Spn {message.location.span})"
-                result += linesep
+            if type(message) is ExecutionMessage:
+                result += message.message + linesep
+                if message.source is not None:
+                    result += f"  Source: {message.source.uri}"
+                    if message.location is not None:
+                        result += f" (Ln {message.location.line}: Col {message.location.column}: Pos {message.location.position}: Spn {message.location.span})"
+                    result += linesep
+            else:
+                result += message + linesep
```

Comment:
It seemed like the purpose was to be able to pass in strings as messages as well, which would cause this method to fail.  Is that not the intended functionality?

Reply:
So, strings can be added to messages at lower levels, but once it has made it up to here, it should have been put in the proper format of being an `ExcecutionMessage` object. If it has not been properly formatted before now, then that is a gap in the data handling. I believe we would want it to fail, but can have handling for that failure of returning a notice to the command line of something like "Internal handling error of messages not being properly formatted"

Reply:
I made it return an error if a message that isnt an ExecutionMessage is passed through.  Is that good?

Reply:
Yeap, I had seen that modification and the tweak to the test to reflect.